test/libsolidity/syntaxTests/types/address/literal_to_payable_address.sol

```diff
@@ -1,8 +1,8 @@
 contract C {
     function f() public pure {
-        address payable a = address(0);
-        a = address(1);
+        address payable a = payable(address(0));
+        a = payable(address(1));
         address payable b = 0x0123456789012345678901234567890123456789;
```

Comment:
Don't we also want to implement https://github.com/ethereum/solidity/issues/6452#issuecomment-574706133 ?

Reply:
How do you want the syntax to look like, in this case?  `address payable b = payable(0x0123456789012345678901234567890123456789)`?

Reply:
I guess so, yes.

Reply:
The PR disallows `payable(literal)`. Instead, it has to go through `payable(address(literal))`. Because of that, this would be `payable(address(0x0123456789012345678901234567890123456789))`.

Do you think disallowing `payable(literal)` should be done?

Reply:
If the literal has a proper address checksum, it should not be necessary to convert to address - is that the case?